_data/materials.yaml

```diff
@@ -3,44 +3,16 @@
 - id: 0
   name: Info Session
   slides: https://docs.google.com/presentation/d/1T3x-K1YJEyDw8fUEX9nsrEkJ4BXwW7d-yffKF0Mb5VE/edit?usp=sharing
```

Comment:
I feel like these slides links should be removed, because I imagine we'd want to copy last semester's slides and labs without modifying the originals, so the link will change. I guess these can be kept here for now, but I think just having dummy links for now might make it clearer that the slides don't actually exist yet.